Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ka.properties:
```diff
@@ -90,7 +90,7 @@ passwordless=უპაროლო
 recovery-authn-code=ჩემი აღდგენის ავთენტიკაციის კოდები
 webauthn-help-text=გამოიყენეთ თქვენი Passkey შესასვლელად.
-error-invalid-multivalued-size=ატრიბუტს {{0}} უნდა ჰქონდეს მინიმუმ {{1}} და მაქსიმუმt {{2}} მნიშვნელობა.
+error-invalid-multivalued-size=ატრიბუტს {{0}} უნდა ჰქონდეს მინიმუმ {1} და მაქსიმუმt {2} მნიშვნელობა {2,choice,0#values|1#value|1<values}.
```
@NorwayFun - I see that you took the original text for `{2,choice,0#values|1#value|1<values}` without changes.

The `choice` format you see here is intended to allow different forms for singular or plural. In English this allows to print "0 values, 1 value, 2 values, ...". I don't know about the Georgian language. If there is only one form, you can simplify it as

```suggestion
error-invalid-multivalued-size=ატრიბუტს {0} უნდა ჰქონდეს მინიმუმ {1} და მაქსიმუმt {2} {2,choice,0#მნიშვნელობა,0<მნიშვნელობა}.
```

ahus1 replied:
Thanks!

NorwayFun committed:
Updated translation for Georgian

Language: ka

Updated translation for Georgian

Language: ka

Translated using Weblate (Georgian)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/ka/

Updated translation for Georgian

Language: ka

Co-authored-by: Temuri Doghonadze <temuri.doghonadze@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Temuri Doghonadze <temuri.doghonadze@gmail.com>

herver1971 committed:
Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Updated translation for Spanish

Language: es

Co-authored-by: Hernan Lopez <hernan.lopez@gmail.com>
Signed-off-by: Hernan Lopez <hernan.lopez@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Update translation files

Updated by "Cleanup translation files" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

ahus1 committed:
Updated translation for Georgian

Language: ka

Translated using Weblate (Georgian)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/ka/

Updated translation for Georgian

Language: ka

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

NorwayFun committed:
Updated translation for Georgian

Language: ka

Updated translation for Georgian

Language: ka

Updated translation for Georgian

Language: ka

Updated translation for Georgian

Language: ka

Co-authored-by: Temuri Doghonadze <temuri.doghonadze@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Temuri Doghonadze <temuri.doghonadze@gmail.com>

robson90 committed:
Updated translation for German

Language: de

Co-authored-by: Robin <39960884+robson90@users.noreply.github.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Robin <39960884+robson90@users.noreply.github.com>

weblate committed:
Updated translation for Dutch

Language: nl

Co-authored-by: Anonymous <noreply@weblate.org>
Signed-off-by: Anonymous <noreply@weblate.org>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

ahus1 commented:
@robson90: The German changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@herver1971 / @anthieni: The Spanish changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@NorwayFun - thank you for adding this many translations!
